### DISCUSSION TITLE
Can't find wfb-cli in OpenIPC gs software?

### DISCUSSION THREAD

**Discussion by lida2003** [svpcom/wfb-ng#338] [Questions] [answered]
How can get wfb-cli command? custom build an OpenIPC?

Any links about build OpenIPC software is well appreciated. As I know it's from buildroot, bla...bla....

![图片](https://github.com/svpcom/wfb-ng/assets/1296171/6ad8f71a-5182-4619-acc6-46d3b7f2734b)

**Comment by svpcom** [comment, marked as answer]
OpenIPC use only subset of wfb-ng. It use wfb_rt/tx (data plane) but not control plane (written in python) and wfb-cli.
If you need a full-featured wfb-ng then use openipc as camera only (i.e. flash **lite** or **ultimate**, not **fpv** firmware) and connect it via ethernet to SBC (RPI, rockchip, allwinner, etc) which will run wfb-ng.

**Reply by lida2003** [reply]
I see, thanks for the info. Maybe I will try to take a look at how to comile OpenIPC firmware section, I don't know where to look right now :(

**Reply by svpcom** [reply]
Try to ask in their telegram channel:  https://t.me/+BMyMoolVOpkzNWUy
